src/components/PersonLink.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+interface PersonLinkProps {
+  person: Person | null;
+}
+
+export const PersonLink: React.FC<PersonLinkProps> = ({ person }) => {
+  return person ? (
+    <Link
+      to={`../${person.slug}`}
+      className={person.sex === 'f' ? 'has-text-danger' : ''}
+    >
+      {person.name}
+    </Link>
+  ) : (
+    <span>-</span>
```

Comment:
it can be just a string (anyway it is in `td` tag). But do you really need this ternary here? - You can make person obligate prop (remove `| null`). For mother and father you anyway check it outside of the PersonLink component